spinalcordtoolbox/gui/sagittal.py

```diff
@@ -20,8 +20,12 @@
 
 
 class SagittalController(base.BaseController):
-    def __init__(self, image, params, init_values=None):
+    def __init__(self, image, params, init_values=None,previous_point=None):
```

Comment:
could you please add a space after "," (PEP8)

Reply:
Yes will do.